src/viam/sdk/module/service.cpp

```diff
@@ -4,8 +4,6 @@
 #include <memory>
 #include <sstream>
 #include <string>
-#include <sys/socket.h>
```

Comment:
I think these were only needed for the creation of the socket below, which we are no longer doing.